modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -29,6 +29,13 @@ ifeval::[{uyuni-content} == true]
 endif::[]
 
 
+[IMPORTANT]
+====
+Traditional clients are not available on {rhel}{nbsp}8.
+{rhel}{nbsp}8 clients are only supported as Salt clients.
+====
+
+
```

Comment:
This is not enough. We need to tell the users how to add Appstream repository and channel. And I am pretty sure the paths for os are now BaseOS. Ornela should now.